frontend/views/containers/proposals/ProposalsWidget.vue

```diff
@@ -97,14 +108,41 @@ export default ({
       // this.proposalsSorted = sortByNotVoted // eslint-disable-line vue/no-side-effects-in-computed-properties
 
       this.proposalsGrouped = proposalsGrouped // eslint-disable-line vue/no-side-effects-in-computed-properties
-
-      return this.proposalsGrouped
+      console.log(this.proposalsGrouped.flat())
```

Comment:
Stray debugging, just a reminder to not forget to remove this or convert to `console.debug` if needed.